docker/base/Dockerfile

```diff
@@ -1,9 +1,9 @@
 ARG cert_image
 ARG root_image
```

Comment:
use ARG VARIABLE_NAME=default_value

Reply:
and let's use 3.19.0, not 3.19.1, this will give us a chance to test dependabot upgrading it to 3.19.1